### PR TITLE
issuers page

### PR DESCRIPTION
**Pull request by sounak98**

### CODE REVIEW

File: src/components/AltairHero/index.js [hunk 1]
```diff
@@ -40,9 +40,10 @@ const AltairHero = () => {
             <Button
               primary
               color="altair"
-              label="Stake Kusama"
+              label="Stake Now"
               alignSelf="start"
               href="https://centrifuge.io/altair/crowdloan"
+              target="_blank"
```
Comment (armandomistral) [inline src/components/AltairHero/index.js:46]: Don't reintroduce target blank – the page should load in the same window…

Comment (sounak98) [reply src/components/AltairHero/index.js:46]: Yeah, just noticed the new commits and have removed it. This PR is WIP, so don't merge it before the Altair Staking release.

File: src/helpers/subgraph.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+async function getData() {
+  const res = await fetch(process.env.GATSBY_CENTRIFUGE_SUBGRAPH_URL, {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({
+      query: `
+              query {
+                pools {
+                  reserve, assetValue
```
Comment (offerijns) [inline src/helpers/subgraph.js:13]:
```suggestion
                  reserve, assetValue, totalBorrowsAggregatedAmount
```

File: src/helpers/subgraph.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  }, new BN(0));
+
+  return tvl;
+}
+
+async function getTaf() {
+  if (!cachedData) cachedData = await getData();
+
+  const taf = cachedData.pools.reduce((sum, pool) => {
+    return sum.add(new BN(pool.assetValue));
```
Comment (offerijns) [inline src/helpers/subgraph.js:40]:
```suggestion
    return sum.add(new BN(pool.totalBorrowsAggregatedAmount));
```

**Review by offerijns — CHANGES_REQUESTED** [review]

**Comment by sounak98** [comment]
@Dy-Dedi I have updated the file, looks much better now

**Pull request merged**